### PR TITLE
guesses are unique and looks more presentable for video

### PR DESCRIPTION
**Pull request by Jopce**
@Tronikelis 

Hello, sir. I would appreciate if you took some time and verified this PR 🙏

### CODE REVIEW

**Comment by Jopce** [comment]
Adding random words from .txt in this PR also.

File: Main.java [hunk 1]
```diff
@@ -31,11 +31,21 @@ private Character getChar() throws IOException {
             String input = stdinReader.readLine().trim();
 
             if (input.length() == 1) {
+                if (guesses.contains(input.charAt(0))) {
+                    System.out.println("This character was already guessed! Try another one.");
+                    printGuesses();
+                    printBoard();
+//                    ch = '\0';
```
Comment (tronikelis) [inline Main.java:38]: ??

File: Main.java [hunk 2]
```diff
@@ -66,6 +76,7 @@ private void printBoard() {
 
             System.out.printf(" ", ch);
         }
+        System.out.println("");
```
Comment (tronikelis) [inline Main.java]: `System.out.println()` :nerd_face:  ??

**Review by tronikelis — APPROVED** [review]

**Comment by tronikelis** [comment]
> Adding random words from .txt in this PR also.
do it in different one, easier to review @Jopce

File: Main.java (hunk 1 above)
Comment (Jopce) [reply Main.java:38]: Checks if guess is unique and reprints the board so it looks clean.

File: Main.java (hunk 2 above)
Comment (Jopce) [reply Main.java]: bruh

File: Main.java (hunk 1 above)
Comment (tronikelis) [reply Main.java:38]: i mean the comment

**Pull request merged**